docs/index.md

```diff
@@ -1,17 +1,32 @@
-# Welcome to MkDocs
+# PHIS developpers documentation
 
-For full documentation visit [mkdocs.org](http://mkdocs.org).
+## Background
+Phis (Phenotyping Hybrid Information System) is developped for high-throughput plant phenotyping in close interaction to plant phenotyping communities.
+Phis is part of SILEX collaborative meta-project developped at MISTEA joint research unit (INRA - SupAgro).
+More information on SILEX is availbale at [SILEX wiki](https://mulcyber.toulouse.inra.fr/plugins/mediawiki/wiki/silex/index.php/Accueil "SILEX wiki Main Page").
+The present documentation is a resource material related to the Phis version developped for field phenotyping, which complete name is SILEX-PHIS-FIELD.
```

Comment:
Please says: The present documentation is a resource material related to the new version of Phis